DogScepterLib/Core/Chunks/GMChunkTGIN.cs

```diff
@@ -57,6 +57,30 @@ private void DoFormatCheck(GMDataReader reader)
 
                 reader.Offset = returnTo;
             }
+
+            if (reader.VersionInfo.IsVersionAtLeast(2022, 9) && !reader.VersionInfo.IsVersionAtLeast(2023, 1))
+            {
+                int returnTo = reader.Offset;
+                reader.Offset += 4; // Skip count.
+
+                uint firstPtr = reader.ReadUInt32();
+
+                // Navigate to the fourth list pointer, which is different
+                // depending on whether this is 2023.1+ or not (either "FontIDs"
+                // or "SpineSpriteIDs").
+                reader.Offset = (int)(firstPtr + 16 + (sizeof(uint) * 3));
```

Comment:
Hmm... honestly unsure if sizeof() is a good idea to be using here, as a constant 4 bytes is applied everywhere else in the project (including in this code). Can probably just change that to 4, but unsure.

Reply:
Doesn't really matter, suppose it's just a difference in codestyle -- it gets compiled directly to four bytes anyway, treated as a constant value.